.gitignore

```diff
@@ -26,3 +26,6 @@ config.js
 
 # builds
 dist
+
+#311
+.idea/*
```

Comment:
Sorry, my suggestion didn't work. I think you can change this to `.idea/` (i.e., get rid of the *). Also, I don't think the comment above ("#311") is useful.